graylog2-server/src/main/java/org/graylog2/indexer/cluster/Cluster.java

```diff
@@ -162,16 +162,11 @@ public boolean isConnected() {
                 .timeout(Ints.saturatedCast(requestTimeout.toSeconds()))
                 .build();
 
-        try {
-            final JestResult result = jestClient.execute(request);
-            final int numberOfDataNodes = Optional.of(result.getJsonObject())
-                .map(json -> asInteger(json.get("number_of_data_nodes")))
-                .orElse(0);
-            return result.isSucceeded() && numberOfDataNodes > 0;
-        } catch (IOException e) {
-            LOG.warn("Couldn't check connection status of Elasticsearch", e);
-            return false;
-        }
+        final JestResult result = JestUtils.execute(jestClient, request, () -> "Couldn't check connection status of Elasticsearch");
+        final int numberOfDataNodes = Optional.of(result.getJsonObject())
+            .map(json -> asInteger(json.get("number_of_data_nodes")))
+            .orElse(0);
+        return result.isSucceeded() && numberOfDataNodes > 0;
```

Comment:
`result.isSucceeded()` is always true, since `JestUtils.execute(...)` would throw an `ElasticsearchException` if it wasn't.